contrib/kafka/pkg/apis/messaging/v1alpha1/kafka_channel_types.go

```diff
@@ -75,7 +75,12 @@ type KafkaChannelStatus struct {
 	// provided targets from inside the cluster.
 	//
 	// It generally has the form {channel}.{namespace}.svc.{cluster domain name}
+	// To do: This should be defined in duck APIs as AddressStatus and then inlined here.
```

Comment:
nit: TODO instead of To do

Reply:
Shouldn't this be duckv1alpha1.AddressStatus embeded here? Similar to subscribableTypeStatus? That way it is defined in one single place. However AddressStatus is defined as
type AddressStatus struct {
	Address *Addressable `json:"address,omitempty"`
}
Which makes Address a pointer, but that can be changed if needed.